Godeps/Godeps.json

```diff
@@ -2551,6 +2551,10 @@
 			"Rev": "55146ba61254fdb1c26d65ff3c04bc1611ad73fb"
 		},
 		{
+			"ImportPath": "google.golang.org/api/logging/v2beta1",
```

Comment:
I know. Bumping deps is no easy that why I suggested to do it in a follow up PR;)